### DISCUSSION TITLE
Should be some math functions like powf() optimized by hands?

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#2097] [General]
During the review of DrawNode V2 comes this questions:
Should be some math functions like powf() optimized by hands?

![image](https://github.com/user-attachments/assets/6ce51df6-9cf3-4cfc-9deb-2f5a4037c248)

**Comment by halx99** [comment]
I think `powf` is ok

**Reply by halx99** [reply]
recommended to use `std::pow`

**Reply by halx99** [reply]
works on linux:
```cpp
#include <cmath>

constexpr int result = std::pow(10.0f, 3);
int main(int argc, char** argv)
{
    char buf[result];
    constexpr auto xxx = sizeof(buf);
    return 0;
}
```
